awscli/customizations/emr/createdefaultroles.py

```diff
@@ -202,19 +202,12 @@ def _run_main(self, parsed_args, parsed_globals):
 
         emrutils.display_response(
             self._session,
-            self._session.get_service('iam').get_operation('CreateRole'),
+            'create_role',
             self._construct_result(ec2_result, emr_result),
             parsed_globals)
 
         return 0
 
-    def _check_for_iam_endpoint(self, region, iam_endpoint):
```

Comment:
This method did not really any sense to me especially, since that botocore always gives the same iam endpoint (no matter the region or even if no region was provided) and UnknownEndpointError will never be thrown.
 
Do you think it is alright that I just removed the check.